concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1085,14 +1085,18 @@ jobs:
                   . ./config/config.sh
                   echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
 
-                  . ./config/quota_settings.sh
-                  ${QUOTA_non_basic_services_allowed} || prefix="dis"
-
-                  cf update-quota "${QUOTA_DEFAULT}" \
-                    -m "${QUOTA_memory_limit}"M \
-                    -r "${QUOTA_total_routes}" \
-                    -s "${QUOTA_total_services}" \
-                    --${prefix}allow-paid-service-plans
+                  for quota_file in config/*_quota.sh; do
+                    (
```

Comment:
+1 for setting `set -u`

But I am not sure about exporting the subshell to script. that means create a script for just a pair of lines that are called from a for. If we do so we are breaking the flow in the loop, and it will it makes it more difficult to understand how everything works together.

If we are worried about the risk of overlooking the subshell, we can add a comment.